external/contracts/aave/AaveTokenV2Mintable.sol

```diff
@@ -0,0 +1,1340 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
Can you also import the other external contracts for reference? E.g. GovernanceStrategy, Executor etc